### ISSUE TITLE
Loss of A2C algorithm

### ISSUE THREAD

**Issue by yingchengyang** [thu-ml/tianshou#21]
I'd like to ask about the loss of a2c algorithm. There are three part of loss in tianshou's code:
                      loss = a_loss + self._w_vf * vf_loss - self._w_ent * ent_loss
I'd like to ask about the meaning of the third part of the loss. I try use this to run in CartPole-v0, it converge after 1355 steps,then I try use only the first and second part of loss to run in CartPole-v0, it converge after 770 steps. It seems that only use the first and second part of loss is enough?

**Comment by Trinkle23897** [comment]
The third loss is about the policy entropy loss. In other repo such as openai baseline, their A2C/A3C implementation has this term, so I add the entropy loss here.
Of course you can set the weight of entropy loss to zero :)

**Comment by yingchengyang** [comment]
thanks！
